### PR TITLE
Adds explanations for in-model errors. Closes 590.

### PR DESCRIPTION
**Pull request by mmcdermott**

### CODE REVIEW

File: src/components/distributions/editor/style.css [hunk 1]
```diff
@@ -4,6 +4,15 @@
   @mixin small-input;
 }
 
+.Guesstimate .metric-errors-tooltip.metric-errors-tooltip.metric-errors-tooltip {
+  text-align: center;
+  opacity: 1;
+  padding: 4px 13px;
+  border-radius: 2px;
+  transition: none;
+  z-index: 10000000 !important;
```
Comment (OAGr) [inline src/components/distributions/editor/style.css]: I don't think this is still needed

File: src/components/metrics/card/MetricCardViewSection/index.js [hunk 2]
```diff
@@ -9,23 +9,36 @@ import StatTable from 'gComponents/simulations/stat_table/index'
...
-const ErrorSection = ({errors, padTop, hide}) => (
-  <div className={`StatsSectionErrors ${isBreak(errors) ? 'minor' : 'serious'} ${padTop ? 'padTop' : ''} ${hide ? 'isHidden' : ''}`}>
-    {isBreak(errors) && <Icon name='unlink'/>}
-    {!isBreak(errors) && isInfiniteLoop(errors) && <i className='ion-ios-infinite'/>}
-    {!isBreak(errors) && !isInfiniteLoop(errors) && <Icon name='warning'/>}
+const ErrorIcon = ({errors}) => {
+  if (isBreak(errors)) { return <Icon name='unlink'/> }
+  else if (isInfiniteLoop(errors)) { return <i className='ion-ios-infinite'/> }
+  else { return <Icon name='warning'/> }
+}
```
Comment (OAGr) [inline src/components/metrics/card/MetricCardViewSection/index.js]: is this used?

**Comment by OAGr** [comment]
Minor comments, mostly with regards to work that wasn't eventually used.

File: src/components/metrics/card/MetricCardViewSection/index.js (hunk 2 above)
Comment (mmcdermott) [inline src/components/metrics/card/MetricCardViewSection/index.js]: Yes. Just a few lines down.

Comment (OAGr) [inline src/components/metrics/card/MetricCardViewSection/index.js]: Oh, got it, thought it was part of the previous not-used version, sorry.

**Pull request merged**